src/constant/enums.js

```diff
@@ -200,3 +234,73 @@ export const PaymentTypeLabels = [
 export function getPaymentTypeLabel(paymentType) {
     return getEnumLabel(PaymentTypeLabels, paymentType);
 }
+
+export const ParkingSize = Object.freeze({
```

Comment:
Same for this too. 

https://github.com/parkspot-dev/maya/blob/master/core/models/struct.go#L200